res/css/views/dialogs/_LeaveSpaceDialog.scss

```diff
@@ -19,58 +19,56 @@ limitations under the License.
         display: flex;
         flex-direction: column;
         padding: 24px 32px;
-    }
-}
 
-.mx_LeaveSpaceDialog {
-    width: 440px;
-    display: flex;
-    flex-direction: column;
-    flex-wrap: nowrap;
-    height: 520px;
+        .mx_LeaveSpaceDialog {
+            width: 440px;
+            display: flex;
+            flex-direction: column;
+            flex-wrap: nowrap;
+            height: 520px;
 
-    .mx_Dialog_content {
-        flex-grow: 1;
-        margin: 0;
-        overflow-y: auto;
+            .mx_Dialog_content {
+                flex-grow: 1;
+                margin: 0;
+                overflow-y: auto;
 
-        .mx_LeaveSpaceDialog_section_warning {
-            position: relative;
-            border-radius: 8px;
-            margin: 12px 0 0;
-            padding: 12px 8px 12px 42px;
-            background-color: $header-panel-bg-color;
+                .mx_LeaveSpaceDialog_section_warning {
+                    position: relative;
+                    border-radius: 8px;
+                    margin: 12px 0 0;
+                    padding: 12px 8px 12px 42px;
+                    background-color: $header-panel-bg-color;
 
-            font-size: $font-12px;
-            line-height: $font-15px;
-            color: $secondary-content;
+                    font-size: $font-12px;
+                    line-height: $font-15px;
+                    color: $secondary-content;
 
-            &::before {
-                content: '';
-                position: absolute;
-                left: 10px;
-                top: calc(50% - 8px); // vertical centering
-                height: 16px;
-                width: 16px;
-                background-color: $secondary-content;
-                mask-repeat: no-repeat;
-                mask-size: contain;
-                mask-image: url('$(res)/img/element-icons/room/room-summary.svg');
-                mask-position: center;
-            }
-        }
+                    &::before {
+                        content: '';
+                        position: absolute;
+                        left: 10px;
+                        top: calc(50% - 8px); // vertical centering
+                        height: 16px;
+                        width: 16px;
+                        background-color: $secondary-content;
+                        mask-repeat: no-repeat;
+                        mask-size: contain;
+                        mask-image: url('$(res)/img/element-icons/room/room-summary.svg');
+                        mask-position: center;
+                    }
+                }
 
-        > p {
-            color: $primary-content;
-        }
-    }
-
-    .mx_Dialog_buttons {
-        margin-top: 20px;
```

Comment:
Removed based on this: https://github.com/matrix-org/matrix-react-sdk/blob/2ab8b46ba3fcac15f66b0c040d7a822aa9ae882e/res/css/_common.scss#L411